web-client/src/presenter/actions/setDefaultTableSortAction.js

```diff
@@ -10,14 +10,13 @@ import { state } from 'cerebral';
  * @param {object} providers.store the cerebral store object
  */
 export const setDefaultTableSortAction = ({ get, props, store }) => {
-  // Story 9490 requires the sorting to only apply to the ADC Section boxes.  So these if statements exist to limit the scope for now.  These if statements can be removed in future stories that expand the scope.
   const userRole = get(state.user.role);
-  if (props.queue !== 'section' || userRole !== 'adc') {
+  if (userRole !== 'adc') {
     store.unset(state.tableSort);
     return;
   }
 
-  // different tables require different sorting
+  // different tables require different default sorting
```

Comment:
Nit: y'all added some really awesome tests for this functionality, is this comment needed?

Reply:
Made the change.